nevergrad/functions/base.py

```diff
@@ -214,6 +215,26 @@ def evaluation_function(self, *args: tp.Any, **kwargs: tp.Any) -> float:
         ), "evaluation_function can only be called on monoobjective experiments."
         return output
 
+    def pareto_evaluation_function(self, *pareto: p.Parameter) -> float:
```

Comment:
@teytaud you should be able to provide all the functionalities you want through this function

Reply:
Not sure, ideas for discussion:
- should this be in MultiExperiment rather than in ExperimentFunction ?
- should ExperimentFunction be named MultiExperimentFunction as the current name leads to a confusion between experiments and experimentfunctions ?

Reply:
So far:
- `MultiExperiment`: a helper to create a multi objective experiment through several mono-objective experiments.
- `ExperimentFunction`: the class which provides the API for all experiments, mono *and* multiobjective functions.

I'd rather we keep one single API for all cases, because they do not differ that much except at evaluation time.
In practice I am rather considering renaming `pareto_evaluation_function` to just `evaluation_function` which will work in all cases: either it gets one single `Parameter` (mono-objective) or the whole pareto (multiobjective). That would just require changing all currently evaluation function to take as input a Parameter instead of args and kwargs. That adds minor complexity but adds a lot of flexibility